src/book/02-system/02-subsystems/04-motion.mdx

```diff
@@ -6,17 +6,70 @@ description: Motion and how it works in the NUbots codebase.
+
+The specific `Spline` type we use is a `SmoothSpline`, extended from the `Spline` class. Find out more about splines and this specific type of spline on the [**mathematics**](/system/foundations/mathematics#splines) page.
+
+The `TrajectoryUtils` file uses the `SplineContainer` for walking and kicking. It creates a `SplineContainer` with the following splines:
+
+| Name                 | Description                                                                              |
+| -------------------- | ---------------------------------------------------------------------------------------- |
+| IS_DOUBLE_SUPPORT    | Splines for when the robot is in double support phase, i.e. both feet are on the ground. |
+| IS_LEFT_SUPPORT_FOOT | Splines for the support foot.                                                            |
+| TRUNK_POS_X          | The Cartesian position of the robot's torso in the x-axis.                               |
+| TRUNK_POS_Y          | The Cartesian position of the robot's torso in the y-axis.                               |
+| TRUNK_POS_Z          | The Cartesian position of the robot's torso in the z-axis.                               |
+| TRUNK_AXIS_X         | The orientation - x-axis of the robot's torso.                                           |
+| TRUNK_AXIS_Y         | The orientation - y-axis of the robot's torso.                                           |
+| TRUNK_AXIS_Z         | The orientation - z-axis of the robot's torso.                                           |
+| FOOT_POS_X           | The Cartesian position of the robot's flying foot in the x-axis.                         |
+| FOOT_POS_Y           | The Cartesian position of the robot's flying foot in the y-axis.                         |
+| FOOT_POS_Z           | The Cartesian position of the robot's flying foot in the z-axis.                         |
+| FOOT_AXIS_X          | The orientation - x-axis of the robot's flying foot.                                     |
+| FOOT_AXIS_Y          | The orientation - y-axis of the robot's flying foot.                                     |
+| FOOT_AXIS_Z          | The orientation - z-axis of the robot's flying foot.                                     |
+
+The points that make-up all these splines are in the [WalkEngine.cpp file](https://github.com/NUbots/NUbots/blob/ff2cf1f7f38bf558494e305a99544e15affaf719/module/motion/QuinticWalk/src/engine/WalkEngine.cpp#L331). The [configuration parameters](https://github.com/NUbots/NUbots/blob/biddulph/quintic_walk/module/motion/QuinticWalk/data/config/QuinticWalk.yaml) affect some of these points. Values are all in [SI units](https://en.wikipedia.org/wiki/International_System_of_Units).
+
+The motion subsystem contains modules responsible for walking, kicking, moving the head, and running scripts. They perform these motions by sending servo commands (called targets) to the motors.
```

Comment:
This paragraph seems a little disjoint. Perhaps this should be at the start of the page somewhere?

Reply:
It's a merge issue, didn't notice it down there.